### PR TITLE
fix: fix rlp circuit bug for deposit tx

### PR DESCRIPTION
**Pull request by chokobole**
# Description

This PR fixes RLP circuit bug for deposit tx.

### CODE REVIEW

File: zkevm-circuits/src/rlp_circuit.rs [hunk 1]
```diff
@@ -1222,11 +1222,6 @@ impl<F: Field> RlpCircuitConfig<F> {
                         meta.query_advice(rlp_table.data_type, Rotation::next()),
                         RlpDataType::TxHash.expr(),
                     );
-                    cb.require_equal(
-                        "TxHash rows' first row is Prefix again",
-                        meta.query_advice(rlp_table.tag, Rotation::next()),
-                        RlpTxTag::Prefix.expr(),
-                    );
```
Comment (dongchangYoo) [inline zkevm-circuits/src/rlp_circuit.rs]: Please restore the deleted code and ensure it only runs when the "kroma" feature is deactivated.

File: zkevm-circuits/src/rlp_circuit.rs [hunk 2]
```diff
@@ -1235,6 +1230,21 @@ impl<F: Field> RlpCircuitConfig<F> {
...
+                ]),
+                |cb| {
+                    cb.require_equal(
+                        "TxHash rows' first row is Prefix again",
+                        meta.query_advice(rlp_table.tag, Rotation::next()),
+                        RlpTxTag::Prefix.expr(),
+                    );
+                },
+            );
+
```
Comment (dongchangYoo) [inline zkevm-circuits/src/rlp_circuit.rs:1256]: Please make this code only runs when the "kroma" feature is activated.

**Review by dongchangYoo — CHANGES_REQUESTED** [review]

File: zkevm-circuits/src/rlp_circuit.rs (hunk 1 above)
Comment (chokobole) [reply zkevm-circuits/src/rlp_circuit.rs:1229]: This is not equivalent to my change. For the non-deposit tx, the validation should be done.

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Pull request merged**